docs/packaging-operator.md

```diff
@@ -147,6 +147,52 @@ You can verify the generated bundle metadata for semantic correctness with the t
 operator-sdk bundle validate /tmp/my-operator-2.0.0-bundle/ --select-optional name=operatorhub
 ```
 
+#### Dockerfile generated from annotations
+
+Our pipeline is generating `Dockerfile` that is used to build bundle image. Exiting `Dockerfile` or `bundle.Dockerfile` is ignored. Generated `Dockerfile` will take label information from `annotations.yaml` file. For example 
+```
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: global-load-balancer-operator
+  operators.operatorframework.io.bundle.channels.v1: alpha
+  operators.operatorframework.io.bundle.channel.default.v1: alpha
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.4.0+git
+  operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v3
+
+  # Annotations for testing.
+  operators.operatorframework.io.test.mediatype.v1: scorecard+v1
+  operators.operatorframework.io.test.config.v1: tests/scorecard/
+
+```
+will generate `Dockerfile`
+
+```
+FROM scratch
+
+# from metadata/annotations.yaml
+LABEL operators.operatorframework.io.bundle.mediatype.v1="registry+v1"
+LABEL operators.operatorframework.io.bundle.manifests.v1="manifests/"
+LABEL operators.operatorframework.io.bundle.metadata.v1="metadata/"
+LABEL operators.operatorframework.io.bundle.package.v1="global-load-balancer-operator"
+LABEL operators.operatorframework.io.bundle.channels.v1="alpha"
+LABEL operators.operatorframework.io.bundle.channel.default.v1="alpha"
+LABEL operators.operatorframework.io.metrics.mediatype.v1="metrics+v1"
+LABEL operators.operatorframework.io.metrics.builder="operator-sdk-v1.4.0+git"
+LABEL operators.operatorframework.io.metrics.project_layout="go.kubebuilder.io/v3"
+LABEL operators.operatorframework.io.test.mediatype.v1="scorecard+v1"
+LABEL operators.operatorframework.io.test.config.v1="tests/scorecard/"
+
+COPY ./manifests manifests/
+COPY ./metadata metadata/
+COPY ./tests/scorecard/ tests/scorecard/
+```
+!!! note
+    We check if directory `tests/scorecard/` specified in `operators.operatorframework.io.test.config.v1` exists. If not, we fail.
```

Comment:
```suggestion
    If you specify the `operators.operatorframework.io.test.config.v1` to embed [scorecard](https://sdk.operatorframework.io/docs/advanced-topics/scorecard/scorecard/) tests in your bundle, make sure the supplied directory path (e.g. `tests/scorecard/` relative from the bundle root directory) actually exists, otherwise the validation will fail
```